app/main_dev/ipc.js

```diff
@@ -201,6 +231,163 @@ export const startDcrlnd = async (
   }
 };
 
+export const startDex = async (walletPath, testnet) => {
+  if (GetDexPID()) {
+    logger.log(
+      "info",
+      "Skipping restart of DEX as it is already running " + GetDexPID()
+    );
+    const creds = GetDexCreds();
+    return { wasRunning: true, ...creds };
+  }
+
+  try {
+    const started = await launchDex(walletPath, testnet);
+    return started;
+  } catch (e) {
+    logger.log("error", "error launching dex: " + e);
+    return e;
+  }
+};
+
+export const checkInitDex = async () => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping check of init since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const init = await initCheckDex();
+    return init;
+  } catch (e) {
+    logger.log("error", "error checking init dex: " + e);
+    return e;
+  }
+};
+
+export const initDex = async (passphrase) => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping init since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const init = await initDexCall(passphrase);
+    return init;
+  } catch (e) {
+    logger.log("error", "error init dex: " + e);
+    return e;
+  }
+};
+
+export const loginDex = async (passphrase) => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping login since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const login = await loginDexCall(passphrase);
+    return login;
+  } catch (e) {
+    logger.log("error", "error login dex: " + e);
+    return e;
+  }
+};
+
+export const logoutDex = async () => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping login since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const login = await logoutDexCall();
+    return login;
+  } catch (e) {
+    logger.log("error", "error logout dex: " + e);
+    return e;
+  }
+};
+
+export const createWalletDex = async (
+  assetID,
+  passphrase,
+  appPassphrase,
+  account,
+  rpcuser,
+  rpcpass,
+  rpclisten,
+  rpccert
+) => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping create wallet since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const createWallet = await createWalletDexCall(
+      assetID,
+      passphrase,
+      appPassphrase,
+      account,
+      rpcuser,
+      rpcpass,
+      rpclisten,
+      rpccert
+    );
+    return createWallet;
+  } catch (e) {
+    logger.log("error", "error create wallet dex: " + e);
+    return e;
+  }
+};
+
+export const getFeeDex = async (addr) => {
+  if (!GetDexPID()) {
+    logger.log("info", "Skipping getfee since dex is not runnning");
+    return false;
+  }
+
+  try {
+    const getFee = await getFeeDexCall(addr);
+    return getFee;
```

Comment:
We added a `getdexinfo` that includes fee and the full DEX configuration: https://github.com/decred/dcrdex/pull/1042
I bring it up because it was suggested that we really should inform users of the lot size before they shell out for the registration fee.  Not sure how this might fit into Decrediton's workflow however.